ui/src/kapacitor/containers/KapacitorPage.js

```diff
@@ -148,6 +148,7 @@ class KapacitorPage extends Component {
         source={source}
         addFlashMessage={addFlashMessage}
         exists={exists}
+        locationState={location.state}
```

Comment:
at a glance, it's not intuitive to me, without seeing this line, what `locationState` would be, downstream. two ideas as options are:
1. since we're passing down a single prop, something with more context like `routerLocationState` would be clearer.
1. just pass down the whole `location` object, since that's a construct we are familiar with since we use `react-router`, and just peel off `.state` exactly where it's needed. in that case, you could define `location` in the respective `propTypes` and simple show `state` as what's being consumed by that component, rather than defining the whole shape of `location`.

Reply:
passing down the whole location object